refinery/data_set_manager/management/commands/process_metadata_table.py

```diff
@@ -98,6 +98,12 @@ def add_arguments(self, parser):
             default=False,
             help='flag for whether this data set will be visible to the public'
         )
+        parser.add_argument(
+            '--delimiter',
+            action='store_true',
+            default="tab",
+            help='Delimiter to use for metadata file parsing'
```

Comment:
Do you want plain `action='store'` here?

Also, from your default value, it also looks like you'll look up the actual character from some enumeration?